alacritty/src/event.rs

```diff
@@ -771,6 +771,7 @@ impl<'a, N: Notify + 'a, T: EventListener> input::ActionContext<T> for ActionCon
             // If we had search running when leaving Vi mode we should mark terminal fully damaged
             // to cleanup highlighted results.
             if self.search_state.dfas().is_some() {
+                self.search_state.dfas = None;
```

Comment:
This looks like you want to use `dfas().take().is_some()`?

Reply:
Wow, you're right. I haven't known it. Thank you for pointing it out.